gatts_windows.go

```diff
@@ -49,6 +49,8 @@ func (a *Adapter) AddService(s *Service) error {
 		return err
 	}
 
+	a.gattServiceProvider = serviceProvider
```

Comment:
If I understood correctly each GATT service requires its own `GattServiceProvider`. So if someone adds two services this would only store the latest created.

Users should be able to add and stop the specific service they want. 

I was thinking that a map with the UUID and the service provider could work, but the BLE spec allows UUIDs to be repeated (a problem already found on other ble libraries https://github.com/hbldh/bleak/issues/362).

So maybe `AddService()` should return some handle to the created service to stop it?

Reply:
> So maybe AddService() should return some handle to the created service to stop it?

That sounds like a very reasonable suggestion!